superset-frontend/src/explore/components/controls/TextAreaControl.jsx

```diff
@@ -94,31 +100,44 @@ class TextAreaControl extends Component {
       if (this.props.readOnly) {
         style.backgroundColor = '#f2f2f2';
       }
+      const codeEditor = (
+        <div>
```

Comment:
What is this `div` for?

Reply:
When trying to wrap a tooltip around the `TextAreaEditor`  without a div, the tooltip doesn't land in the right place. Placing a div enables the tooltip to behave as intended.